server/users/service.go

```diff
@@ -39,6 +40,16 @@ func RecognitionPayloadBuilder() interface{} {
 	return &RecognitionPayload{}
 }
 
+func (ei *EventIdentifiers) IsAllProperties() bool {
```

Comment:
It would be more clear to have a func `AllIdentificationValuesFilled` or something like that. `Properties` - is general, and isn't clear enough.

Reply:
Renamed to `IsAllIdentificationValuesFilled`